### PR TITLE
Fixes Requested By Caravel Team

### PR DESCRIPTION
**Pull request by donn**
~ Fixed remove_nets: now preserves pins
~ Disabled Global Routing Timing Optimization By Default

### CODE REVIEW

File: scripts/utils/deflef_utils.tcl [hunk 1]
```diff
@@ -115,15 +115,16 @@ proc merge_components {args} {
...
     {-to required} \
   }
   set flags {}
   parse_key_args "move_pins" args arg_values $options flags_map $flags
-  try_catch $::env(OPENROAD_BIN) -python $::env(SCRIPTS_DIR)/defutil.py merge_pins\
+  try_catch $::env(OPENROAD_BIN) -python $::env(SCRIPTS_DIR)/defutil.py replace_pins\
     --output $arg_values(-to)\
     --input-lef $::env(MERGED_LEF)\
-    $arg_values(-to) $arg_values(-from)
+    $arg_values(-from) $arg_values(-to)
```
Comment (kareefardi) [inline scripts/utils/deflef_utils.tcl:127]: might as well make these options/arguments

**Review by kareefardi — APPROVED** [review]

File: scripts/utils/deflef_utils.tcl (hunk 1 above)
Comment (donn) [reply scripts/utils/deflef_utils.tcl:127]: eh

Comment (kareefardi) [reply scripts/utils/deflef_utils.tcl:127]: like use `--from` and `--to`

Comment (donn) [reply scripts/utils/deflef_utils.tcl:127]: this isn't a "what?" eh it's more of a "meh"

**Pull request merged**